src/envo/envo-edit.owl

```diff
@@ -3535,6 +3553,15 @@ Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510038>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510039>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510040>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510041>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510055>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510056>))
```

Comment:
This should be under `environmental material` or a subclass thereof. Consider `[glue](http://purl.obolibrary.org/obo/ENVO_00003076)` - it's imperfect, but better. 

`methyl cellulose` and `water` are in the axioms, but not mentioned in the textual definition. 
Rephrase the latter to "A glue which is composed primarily of methyl cellulose and water, and used to fix wallpaper to walls." or similar

Reply:
'Wallpaper paste' has been placed as a subclass of glue and the textual definition modified as recommended.